src/main/java/projectzero/cli/CliApplication.java

```diff
@@ -1,122 +1,351 @@
+        String field = s.substring(s.indexOf(" ") + 1);
+
+            try{
+                MainManager.getUmlClass(className).addField(new Field(field));
+                System.out.println("The field " + field + " was added to " + className);
+            }catch (InvalidNameException e){
+                System.out.println("Invalid field input");
+            }catch (NullPointerException e){
+                System.out.println("Class not found");
+            }
+
+    }
+
+    private void addMethod(String s) {
+        String className = s.substring(0, s.indexOf(" "));
+        String method = s.substring(s.indexOf(" ") + 1);
+
+
+             try{
+                 MainManager.getUmlClass(className).addMethod(new Method(method));
+                 System.out.println("Method " + method + " was added to " + className);
+             }catch (InvalidNameException e){
+                 System.out.println("Invalid method input");
+             }catch (NullPointerException e){
+                 System.out.println("Class not found");
+             }
+
+
+    }
+
+    private void printHelp() {
+        File help = new File("src/main/java/projectzero/cli/helpMenu");
+        try{
+            Scanner fileRead = new Scanner(help);
+            while(fileRead.hasNext()){
+                System.out.println(fileRead.nextLine());
+            }
+        }catch(Exception e){
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void addClass(String name){
+
+        try{
+            MainManager.addUmlClass(new UmlClass(name));
+            System.out.println(name + " " +  "was added");
+        }catch (InvalidNameException e){
+                System.out.println("Not a valid class name.");
+            }
+        catch (NullPointerException e){
+            System.out.println("Class already exists.");
+        }
+
+
     }
 
     private void printList() {
         List<UmlClass> tempList = MainManager.listUmlClasses();
         System.out.println("\nCurrent classes:");
-        System.out.print("[ ");
         for (UmlClass tempClass : tempList) {
-            System.out.print(tempClass.getName() + " ");
+            System.out.print("[ ");
+            System.out.println(tempClass.getName() + ": ");
+            System.out.print("\t");
+            System.out.println("Fields: ");
+            for(Field f: tempClass.getFields()){
+                System.out.println("\t  " + f.getName());
+            }
+            System.out.print("\t");
+            System.out.println("Methods: ");
+            for(Method m: tempClass.getMethods()){
+                System.out.println("\t  " + m.getName());
+            }
+            System.out.print("\t");
+            System.out.println("Relationship Points to: ");
+            for(Relationship r: tempClass.getRelationships()){
+                System.out.println("\t  " + r.getTo().getName());
+            }
+            System.out.println("]\n");
```

Comment:
Might be easier to use the StringBuilder class maybe. Just a suggestion for later.